src/movies.js

```diff
@@ -4,10 +4,56 @@
 
 // Get the average of all rates with 2 decimals 
 
+ratesAverage = function(arr){
+  var ratesArr = arr.map((arr) => {return arr.rate});
+
+  var ratesSum = ratesArr.reduce(function(accumulator, currentNum){
+    return accumulator + currentNum;
+  });
+
+  var average = ratesSum / ratesArr.length;
+
+  if(Number.isInteger(average)){
+    return parseInt(average);
+  } else {
+    return Math.round(average * 100) / 100;
+  }
+}
 
 // Get the average of Drama Movies
 
 
+dramaMoviesRate = function(arr){
+
+  var dramaMovies = arr.filter(function(movies){
+    if(movies.genre.includes('Drama')) {
+      console.log(movies.rate)
+      return movies.rate;
+    }
+  })
+  if(dramaMovies.length === 0) return undefined;
```

Comment:
this is inherently falsey so you can just put !dramaMovies.length and it should work :) It makes it shorter